src/esw/brushed_motors/Controller.cpp

```diff
@@ -196,12 +198,42 @@ void Controller::turnOn() const {
 }
 
 // REQUIRES: nothing
-// MODIFIES: isLive
+// MODIFIES: nothing
+// EFFECTS: Returns a combined ID for both the deviceAddress and motorID
+// MotorID can only be max 3 bits (0-5), and device address is max 2 bits (1 or 2)
+uint8_t Controller::combineDeviceMotorID() const {
+    return (deviceAddress << 3) | motorID;
+}
+
+// REQUIRES: nothing
+// MODIFIES: liveMap
 // EFFECTS: I2C bus, if not already live,
 // configures the physical controller.
 // Then makes live.
+
+std::unordered_map<uint8_t, LiveState> Controller::liveMap;
```

Comment:
I'm not sure if you ran into this issue when emplacing but if you run into issues when emplacing, it's probably because C++ does not know how to handle default stuff when it is a uint8_t (it requires a hash). An easy and acceptable fix is to just make the unordered map take an integer fr the key